python/tests/graph/test_pieces.py

```diff
@@ -172,3 +170,5 @@ def test_pipeline(self):
                 # NamedTemporaryFile(prefix="gdef", suffix=".html").name)
 
             self.assertTrue(np.all(preds_tgt == preds_ref))
+
+model_sizes = {'InceptionV3':(299,299), 'Xception':(299,299), 'ResNet50':(224,224)}
```

Comment:
Nit: Spaces after the colons & commas, e.g: 
`model_sizes = {'InceptionV3': (299, 299), 'Xception': (299, 299), 'ResNet50': (224, 224)}`